etc/config.xml

```diff
@@ -22,7 +22,7 @@
                 <regular_checkout_active>1</regular_checkout_active>
                 <hide_proceed_to_checkout>1</hide_proceed_to_checkout>
                 <express_checkout_active>1</express_checkout_active>
-                <style_css><![CDATA[.amwal-express-checkout-button.product-detail-page,  .box-tocart .action.tocart{ width: 70%; } @media (max-width: 767px) { .amwal-express-checkout-button.product-detail-page { width: unset; } .catalog-product-view .hydrated center { text-align: right; background: #fff; position: fixed; bottom: 0; left: 0; z-index: 9999; width: 100%; } .button-text { padding-left: 0 !important; } .products-grid .product-item-actions .amwal-express-checkout-button { width: 90% !important; } } .product-item-actions .actions-primary{ width: 100%; }]]></style_css>
+                <style_css><![CDATA[.amwal-express-checkout-button.product-detail-page,  .box-tocart .action.tocart{ width: 70%; } @media (max-width: 767px) { .amwal-express-checkout-button.product-detail-page { width: unset; } .catalog-product-view .hydrated center { text-align: right; background: #fff; position: fixed; bottom: 0; left: 0; z-index: 9999; width: 100%; } .button-text { padding-left: 0 !important; } .products-grid .product-item-actions .amwal-express-checkout-button { width: 90% !important; } } .product-item-actions .actions-primary{ width: 100%; } .amwal-wrapper { border: 1px solid #dfdfdf; border-radius: 5px; margin-bottom: 25px; } .amwal-wrapper-login { display: flex; justify-content: center; align-items: center; flex-direction: column; } .amwal-or { position: relative; top: 10px; width: 40px; height: 10px; line-height: 0; text-align: center; margin-left: auto; margin-right: auto; color: #757575; background: white; }]]></style_css>
```

Comment:
are you sure you want to put the css only in the default and not in the html template? They can always override it here later.

Reply:
Ok, I will add it as static CSS and if they need to update something they can override